lib/WWW/Mechanize.pm

```diff
@@ -139,14 +139,14 @@ use Tie::RefHash;
 use HTTP::Request 1.30;
 use LWP::UserAgent 6.45;
 use HTML::Form 1.00;
-use HTML::TokeParser;
+use HTML::TokeParser ();
 use Scalar::Util qw(tainted);
 
 use base 'LWP::UserAgent';
 
 our $HAS_ZLIB;
 BEGIN {
-    $HAS_ZLIB = eval 'use Compress::Zlib (); 1;';
+    $HAS_ZLIB = eval {use Compress::Zlib (); 1;};
```

Comment:
If you change eval string to eval block, we should use `require` instead of `use` here, I think.